### ISSUE TITLE
New organization

### ISSUE THREAD

**Issue by chasestarr** [chasestarr/fifty-fifty#1]
Hey @josephbisaillon

Do you mind taking a look at the new organization pattern?

I'm not sure if I'm optimally using module.exports. Take a look at config/db.js for an example of one.

**Comment by josephbisaillon** [comment]
That's actually one of the better organizational patterns. 

If you follow John Papa, that's one of his design patterns.

**Comment by josephbisaillon** [comment]
If you use, webstorm you can use sectional collapsing too. For better visualization, this should help.

**Comment by chasestarr** [comment]
Cool, Do you have a link to an example of one of his projects?

I'm gonna look into webstorm. Supposedly vscode is getting that feature next month

**Comment by josephbisaillon** [comment]
webstorm is free for personal use. Here is his full https://github.com/johnpapa?tab=repositories he's got a website with good blog posts too

**Comment by josephbisaillon** [comment]
He's got a VScode repo set up

**Comment by chasestarr** [comment]
Did you do his angular course on plural sight?

**Comment by josephbisaillon** [comment]
yeah i did his "clean code" course, it was really good.

**Comment by chasestarr** [comment]
Nice, I'm probably gonna sign up for the site

**Comment by josephbisaillon** [comment]
Eh, If I were you I'd go with egghead.io it's better for JS devs.

**Comment by chasestarr** [comment]
interesting I hadn't heard of the site before

**Comment by josephbisaillon** [comment]
shorter more to the point videos. Need to learn a specific ES6 topic, they will have a 4-6 minute video on it. Better than having to run through 6 hours of content all the time to learn something new.

**Comment by chasestarr** [comment]
yeah true. I suppose it depends on how deep you want to go on the topic
